src/components/ImageView/index.spec.tsx

```diff
@@ -34,25 +36,74 @@ describe(__filename, () => {
     );
   });
 
-  it('does not render an img tag when btoa fails', () => {
+  it('throws an error when btoa fails', () => {
+    const message = 'An error message';
     const _btoa = jest.fn().mockImplementation(() => {
-      throw new Error();
+      throw new Error(message);
     });
 
-    const root = render({ _btoa });
-
-    expect(root.find('img')).toHaveLength(0);
+    expect(() => {
+      render({ _btoa });
+    }).toThrow(message);
   });
 
-  it('logs a debug message when btoa fails', () => {
-    const _btoa = jest.fn().mockImplementation(() => {
-      throw new Error();
-    });
+  it('logs an error message if an image has an invalid mimeType', () => {
     const _log = createFakeLogger();
 
-    render({ _btoa, _log });
+    render({ _log, mimeType: 'invalid/type' });
+
+    expect(_log.error).toHaveBeenCalled();
```

Comment:
I don't mind if you keep this test but I really think these type of tests are not helpful. It will just slow down the test suite and carry a maintenance burden. You already have a test to catch that `null` is returned so you will get the code coverage points 🏀